include/solver.h

```diff
@@ -99,46 +99,46 @@ class AbsSmtSolver
    * @return a Sort object
    */
   virtual Sort make_sort(const SortKind sk,
-                         const std::vector<Sort> & sorts,
+                         const SortVec & sorts,
                          const Sort & sort) const = 0;
 
   /* Make a boolean value term
    * @param b boolean value
    * @return a value term with Sort BOOL and value b
    */
-  virtual Term make_value(const bool b) const = 0;
+  virtual Term make_term(const bool b) const = 0;
 
```

Comment:
const bool -> bool

Reply:
Look for other ints/bools with const qualifiers (e.g. make_sort) - these should be removed

Reply:
Thanks for the feedback/reminders! I made the changes in recent commits (looks like you already looked at those).